dll/win32/shell32/stubs.cpp

```diff
@@ -451,26 +451,38 @@ SheSetCurDrive(INT iIndex)
     return 1;
 }
 
-/*
- * Unimplemented
- */
 EXTERN_C LPWSTR
 WINAPI
-SheRemoveQuotesW(LPWSTR lpInput)
+SheRemoveQuotesW(LPWSTR psz)
 {
-    FIXME("SheRemoveQuotesW() stub\n");
-    return NULL;
+    PWCHAR pch;
+
+    if (*psz == L'"')
+    {
+        for (pch = psz + 1; *pch && *pch != L'"'; ++pch)
+            *(pch - 1) = *pch;
+        if (*pch == L'"')
+            *(pch - 1) = UNICODE_NULL;
+    }
+
+    return psz;
 }
 
-/*
- * Unimplemented
- */
 EXTERN_C LPSTR
 WINAPI
-SheRemoveQuotesA(LPSTR lpInput)
+SheRemoveQuotesA(LPSTR psz)
 {
-    FIXME("SheRemoveQuotesA() stub\n");
-    return NULL;
+    PCHAR pch;
+
+    if (*psz == '"')
```

Comment:
```suggestion
    if (psz && *psz == '"')
```

Wouldn't your initial implemenation crash when being called with a nullptr? I would recommend to add a test with NULL being passed also.

Reply:
The original function doesn't check it.

Reply:
Added `NULL` tests.
Confirmed in Win2k3 and WinXP.
Win10 skips the tests.